app/lib/katello/resources/candlepin.rb

```diff
@@ -417,7 +417,10 @@ def imports(organization_name)
 
           def pools(key, filter = {})
             if key
-              json_str = self.get(join_path(path(key), 'pools') + hash_to_query(filter), self.default_headers).body
+              params = hash_to_query(filter)
+              params += params == '?' ? '' : '&'
+              params += 'attribute=unmapped_guests_only:!true'
```

Comment:
That was my initial thought as well but candlepin does not like the escaped ':' and '!' (%3A and %21)

Invalid format for query parameter attribute. Expected format: attribute=name:value